src/main/java/cloud/eppo/api/EppoValue.java

```diff
@@ -130,4 +132,20 @@ public boolean equals(Object otherObject) {
   public int hashCode() {
     return Objects.hash(type, boolValue, doubleValue, stringValue, stringArrayValue);
   }
+
+  private static String joinStringArray(List<String> stringArray) {
+    if (stringArray == null || stringArray.isEmpty()) {
+      return "";
+    }
+    String delimiter = " ,";
```

Comment:
Any idea why we join with `" ,"` instead of `", "`?

Reply:
Good spot! I think it was probably a mistake made earlier; I think we use string arrays for ONE OF checks and don't rely on `toString()` so you can fix this up.

Reply:
✔️